src/config/config.dev.ts

```diff
@@ -23,6 +23,9 @@ const DevConfig: QuaggaJSConfigObject = {
     },
     locate: true,
     numOfWorkers: 0,
```

Comment:
There shouldn't be any numOfWorkers parameters left anywhere, so remove this one